koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -176,11 +175,12 @@ def clean(self):
 
 
 class LabelsWrapper(object):
-    def __init__(self, dockerfile_path, logger_name=None):
+    def __init__(self, dockerfile_path, logger_name=None, labels_override=None):
         self.dockerfile_path = dockerfile_path
         self._setup_logger(logger_name)
         self._parser = None
         self._label_data = {}
+        self._labels_override = labels_override
```

Comment:
My javascript background spills over every now and then :)

Method argument defaults are shared between calls. In most cases, this is not the intention and can cause some nasty bugs - thus immutable values are better suited. My approach here was to default to a value within the function, which creates a new object for every function call.